test/unit/corporate_information_page_test.rb

```diff
@@ -207,6 +207,24 @@ def self.should_be_invalid_without(type, attribute_name)
     corporate_information_page4.update_in_search_index
   end
 
+  test "will be indexed if the org it belongs to is not live on gov.uk and the document is an accessible policy" do
```

Comment:
We could make the intent a little clearer:

```suggestion
  test "will index accessible policy document even if the org it belongs to is not live on gov.uk" do
```

Reply:
Yeah, that's worded much better than what I had written...